docs/Manual_Install.md

```diff
@@ -280,7 +280,7 @@ To install a standalone IMS component/application server, you need to:
 
 The I-CSCF is responsible for sending requests to the correct S-CSCF. It queries the HSS, but if the HSS doesn't have a configured S-CSCF for the subscriber then it needs to select an S-CSCF itself. The I-CSCF defaults to selecting the Clearwater S-CSCF (as configured in `scscf_uri` in `/etc/clearwater/shared/config`).
 
-You can configure what S-CSCFs are available to the I-CSCF by editing the `/etc/clearwater/s-cscf.json` file.
+You can configure what S-CSCFs are available to the I-CSCF by editing the `s-cscf.json` file.
```

Comment:
I think this needs some reference to the cw-config command that you'd use to extract it.